code/game/objects/structures/tables_racks.dm

```diff
@@ -335,6 +391,69 @@
 	log_combat(shover, target, "shoved", "onto [src] (table)")
 	return COMSIG_CARBON_SHOVE_HANDLED
 
+/obj/structure/flippedtable
```

Comment:
I don't think this should be its own object. It makes far more sense for a table being flipped to be part of the status of the table. Composition!

Reply:
Then you don't need all the copying stuff

Reply:
Will do, I'll try to do any other change requests that popped up first though